src/ray/protobuf/serve.proto

```diff
@@ -60,6 +60,14 @@ message AutoscalingConfig {
 
   // The multiplicative "gain" factor to limit downscale.
   optional double downscale_smoothing_factor = 11;
+
+  // The policy can be passed in as a string import path or callable by the user. When
+  // the policy is passed in as a callable, the policy will be just the name of the
+  // function.
+  optional string policy = 12;
+
+  // The cloudpickled policy definition.
+  bytes serialized_policy_def = 13;
```

Comment:
why do both need to be stored in the autoscaling config?

Reply:
oh good point, I think only `serialized_policy_def` is needed for deserializing later on. Let me drop policy